mask_rcnn_model:
        logs_name: logs
        weights_name: mask_rcnn_512_objects2019_0087.h5
        mask_rcnn_configs:
                NAME: 'objects2019'
                USE_MULTIPROCESSING: True
                GPU_COUNT: 1
                IMAGES_PER_GPU: 1
                STEPS_PER_EPOCH: 1000
                VALIDATION_STEPS: 50 
                BACKBONE: 'mobilenetv1'
                COMPUTE_BACKBONE_SHAPE: None
                BACKBONE_STRIDES: [4, 8, 16, 32, 64]
                FPN_CLASSIF_FC_LAYERS_SIZE: 1024
                TOP_DOWN_PYRAMID_SIZE: 256
                NUM_CLASSES: 15
                #RPN_ANCHOR_SCALES: [8, 16, 32, 64, 128]
                RPN_ANCHOR_SCALES: [16, 32, 64, 128, 256]
                RPN_ANCHOR_RATIOS: [0.5, 1, 2]
                RPN_ANCHOR_STRIDE: 1
                RPN_NMS_THRESHOLD: 0.7
                RPN_TRAIN_ANCHORS_PER_IMAGE: 256
                PRE_NMS_LIMIT: 6000
                POST_NMS_ROIS_TRAINING: 2000
                POST_NMS_ROIS_INFERENCE: 1000
                USE_MINI_MASK: True
                MINI_MASK_SHAPE: (56, 56)
                IMAGE_RESIZE_MODE: 'square'
                IMAGE_MIN_DIM: 512
                IMAGE_MAX_DIM: 512
                IMAGE_MIN_SCALE: 0
                MEAN_PIXEL: [123.7, 116.8, 103.9]
                TRAIN_ROIS_PER_IMAGE: 200
                ROI_POSITIVE_RATIO: 0.33
                POOL_SIZE: 7
                MASK_POOL_SIZE: 14
                MASK_SHAPE: [28, 28]
                MAX_GT_INSTANCES: 100
                RPN_BBOX_STD_DEV: [0.1, 0.1, 0.2, 0.2]
                BBOX_STD_DEV: [0.1, 0.1, 0.2, 0.2]
                DETECTION_MAX_INSTANCES: 100
                DETECTION_MIN_CONFIDENCE: 0.7
                DETECTION_NMS_THRESHOLD: 0.3
                LEARNING_RATE: 0.001
                LEARNING_MOMENTUM: 0.9
                WEIGHT_DECAY: 0.0001
                USE_RPN_ROIS: True
                TRAIN_BN: False
                GRADIENT_CLIP_NORM: 5.0
        class_names: ['BG', 'apple', 'cereal', 'chocolate_drink', 'coke', 'crackers', 'grape_juice', 'noodles', 'orange', 'orange_juice', 'paprika', 'potato_chips', 'pringles', 'sausages', 'sprite', 'tray']
